src/m365/spo/commands/listitem/listitem-list.ts

```diff
@@ -227,13 +265,53 @@ class SpoListItemListCommand extends SpoCommand {
         data: requestBody
       };
 
-      const listItemInstances = args.options.camlQuery ? await request.post<ListItemInstanceCollection>(requestOptions) : await request.get<ListItemInstanceCollection>(requestOptions);
-      listItemInstances.value.forEach(v => delete v['ID']);
-      logger.log(listItemInstances.value);
+      const listItemInstances = await request.post<ListItemInstanceCollection>(requestOptions);
+      skipTokenId = listItemInstances.value.length > 0 ? listItemInstances.value[listItemInstances.value.length - 1].Id : undefined;
+      items.push(...listItemInstances.value);
     }
-    catch (err: any) {
-      this.handleRejectedODataJsonPromise(err);
+    while (skipTokenId !== undefined);
+
+    return items;
+  }
+
+  private getExpandFieldsArray(fieldsArray: string[]): string[] {
+    const fieldsWithSlash: string[] = fieldsArray.filter(item => item.includes('/'));
+    const fieldsToExpand: string[] = fieldsWithSlash.map(e => e.split('/')[0]);
+    const expandFieldsArray: string[] = fieldsToExpand.filter((item, pos) => fieldsToExpand.indexOf(item) === pos);
+    return expandFieldsArray;
+  }
+
+  private async getFormDigestValue(logger: Logger, webUrl: string): Promise<string> {
```

Comment:
It seems overkill to create a method that's used in one place and just calls another method and returns one of the properties from the returned object. I suggest we call `spo.getRequestDigest` in place and avoid the unnecessary complexity.